google/client_factory_bigtable.go

```diff
@@ -0,0 +1,22 @@
+package google
+
+import (
+	"context"
+
+	"cloud.google.com/go/bigtable"
+	"golang.org/x/oauth2"
+	"google.golang.org/api/option"
+)
+
+type ClientFactoryBigtable struct {
```

Comment:
I mostly started it with ClientFactory to match the existing Client prefix. Renamed.